### PR TITLE
WIP: Update README with most current developer setup info

### PR DESCRIPTION
**Pull request by seancribbs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,88 @@
...
+   [NetBricks](//github.com/williamofockham/NetBricks) into the same parent
+   directory.
+   ```shell
+   $ for repo in utils MoonGen NetBricks; do \
+       git clone git@github.com:williamofockham/${repo}.git; \
+     done
+   ```
+2. [Install Vagrant](https://www.vagrantup.com/docs/installation/) and
+   [VirtualBox](https://www.virtualbox.org/wiki/Downloads).
+3. Install the `vagrant-disksize` (required) and `vagrant-vbguest` (recommended)
```
Comment (drunkirishcoder) [inline README.md]: I also had to install `vagrant-triggers` as well. I think the docker provisioner needed it? @zeeshanlakhani will know more.

Comment (seancribbs) [reply README.md]: Originally, yes, but we removed the trigger, so you shouldn't need it anymore.

Comment (zeeshanlakhani) [reply README.md]: cool.

**Comment by drunkirishcoder** [comment]
Worth mention that moongen and netbricks repos are mounted at `/MoonGen` and `/NetBricks`.

**Pull request merged**